### PR TITLE
Refactor test steps

### PR DESCRIPTION
**Pull request by tisnik**

### CODE REVIEW

**Comment by tisnik** [comment]
[test]

**Comment by tisnik** [comment]
[test]

File: integration-tests/features/steps/gemini.py [hunk 1]
```diff
@@ -166,8 +173,7 @@ def check_cves_for_epv(context, cves, p, v, e):
         package = check_and_get_attribute(dependency, "name")
         version = check_and_get_attribute(dependency, "version")
         if ecosystem == e and package == p and version == v:
-            assert int(cve_count) == int(cves), \
-                "{exp} CVEs expected, but {found} was found".format(exp=cves, found=cve_count)
+            check_cve_count(cve_count, cves)
```
Comment (miteshvp) [inline integration-tests/features/steps/gemini.py]: There could be a CVE found for EPV later. I guess we should check `CVEs atleast expected but found less` statements

Comment (tisnik) [reply integration-tests/features/steps/gemini.py]: yes @miteshvp, let's change it. TY

**Pull request merged**